app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -117,9 +118,7 @@ class KiwixMainActivity : CoreMainActivity() {
     super.onCreate(savedInstanceState)
     activityKiwixMainBinding = ActivityKiwixMainBinding.inflate(layoutInflater)
     setContentView(activityKiwixMainBinding.root)
-    if (intent.action == "GET_CONTENT") {
-      navigate(R.id.downloadsFragment)
-    }
+    handleShortcutIntent(intent)
```

Comment:
The name of the function should be more clear. Currently, the name sounding it is for handling all shortcuts but it is only handling the `GET_CONTENT` shortcut so the name should be `handleGetContentIntent`.

Reply:
my bad ,done !!